custom_components/eufy_security/const.py

```diff
@@ -288,6 +288,8 @@ class DEVICE_TYPE(Enum):
     SOLO_CAMERA_SPOTLIGHT_1080 = 60
     SOLO_CAMERA_SPOTLIGHT_2K = 61
     SOLO_CAMERA_SPOTLIGHT_SOLAR = 62
+    BATTERY_DOORBELL_DUAL = 91
```

Comment:
I'm not sure this constant name is correct. My doorbell is a wired dual:
```
eufy_security - Exception - process_messages: 91 is not a valid DEVICE_TYPE
```

Reply:
Maybe `DOORBELL_DUAL_8213` (suffix model number) would be interesting to see the model number of the 93 device

Reply:
For more context, my 91 device supports both wired or battery

Reply:
You're right, I had a momentary lapse in sanity and swapped wired and battery device IDs. Let me swap them back.

Reply:
Fixed.  Yeah, it "works" either way, but in the interest of correctness and future-proofing it'd be better to have the device types correctly labeled!